docs/contributing/dev-environment.md

```diff
@@ -18,9 +18,16 @@ If you just want to use Invoke, you should use the [launcher][launcher link].
 
 2. [Fork and clone][forking link] the [InvokeAI repo][repo link].
 
-3. Create an directory for user data (images, models, db, etc). This is typically at `~/invokeai`, but if you already have a non-dev install, you may want to create a separate directory for the dev install.
+3. This repository uses Git LFS to manage large files. To ensure all assets are downloaded:
+      - Install git-lfs → [Download here](https://git-lfs.com/)
+      - Enable automatic LFS fetching for this repository:
+        ```shell
+        git config lfs.fetchinclude "*"
+        ```
 
```

Comment:
```suggestion
        git config lfs.fetchinclude "*"
        ```
        - Fetch files from LFS (only needs to be done once; subsequent `git pull` will fetch changes automatically):
        ```
        git lfs pull
        ```
```